packages/ngtools/webpack/src/transformers/ctor-parameters.ts

```diff
@@ -175,6 +177,15 @@ function typeReferenceToExpression(
       return ts.createIdentifier('Number');
     case ts.SyntaxKind.TypeReference:
       const typeRef = node as ts.TypeReferenceNode;
+      const typeSymbol = typeChecker.getSymbolAtLocation(typeRef.typeName);
```

Comment:
Should we care for aliased symbols?

Reply:
Yes.  That's the failure cause.  Although our unit test setup for the transformers doesn't support the necessary file setup to fully test this.  The E2E test provides coverage though.